src/scrabble.js

```diff
@@ -1,5 +1,67 @@
-function scrabble() {
+const wordValues = {
+    _: 0,
+    A: 1, E: 1, I: 1, O: 1, U: 1, L: 1, N: 1, R: 1, S: 1, T: 1,
+    D: 2, G: 2,
+    B: 3, C: 3, M: 3, P: 3,
+    F: 4, H: 4, V: 4, W: 4, Y: 4,
+    K: 5,
+    J: 8, X: 8,
+    Q: 10, Z: 10
+}
+
+let score = 0
+
+function scrabble(word) {
+    const wordArray = sanitizeAndConvert(word)
+    for (let letter of wordArray) { score += wordValues[letter] }
+    return outputFinalResultAndReset(score)
+}
```

Comment:
Fantastic solution! But let's step away from the one liner ledge, you got a little carried away with it in this exercise

```js
function scrabble(word) {
    const wordArray = sanitizeAndConvert(word)
    
    for (let letter of wordArray) {
        score += wordValues[letter]
    }
    
    return outputFinalResultAndReset(score)
}
```

That takes up more space but it reads like a paragraph in a book, you can glance at it and the eyes know where to look

Reply:
Added white space (though not quite as much) and updated PR.

Reply:
Added new solution based on today's morning session: new_scrabble.js (tests new_scrabble-spec.js).